src/nodes.coffee

```diff
@@ -3242,6 +3279,9 @@ exports.Elision = class Elision extends Base
 
   eachName: (iterator) ->
 
+  ast: ->
+    null
```

Comment:
`Elision`s just generate `null` ASTs in their parent `Arr`'s list of `elements`

Reply:
Yeah, that’s kind of surprising, at least to me. I guess Babel has their reasons, I would’ve expected an `Elision` node type or something.